src/generic/colrdlgg.cpp

```diff
@@ -37,6 +37,19 @@
 #include "wx/colourdata.h"
 #include "wx/generic/colrdlgg.h"
 
+#if USE_PREVIEW_WITH_ALPHA
+#include <wx/statbmp.h>
```

Comment:
We should use quotes and not angle brackets around the includes in wxWidgets itself.